package.json

```diff
@@ -80,7 +79,7 @@
         "mathjax": "^3.1.2"
     },
     "resolutions": {
-        "canvas": "npm:empty-npm-package",
+        "canvas": "npm:empty-npm-package@1.0.0",
```

Comment:
the clue that this might work was in the yarn berry error message that npmjs.com didn't have the tag "empty-npm-package"

I realized it wasn't resolving the parts as we expected - when I added the version to the specification it correctly resolved "empty-npm-package" at it's current version  ref (https://www.npmjs.com/package/empty-npm-package?activeTab=versions) and canvas no longer tried to build and broke on my system - in both yarn versions